staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/naming_controller.go

```diff
@@ -191,26 +191,26 @@ func (c *NamingConditionController) calculateNamesAndConditions(in *apiextension
 		namesAcceptedCondition.Message = "no conflicts found"
 	}
 
-	// set EstablishedCondition to true if all names are accepted. Never set it back to false.
-	establishedCondition := apiextensions.CustomResourceDefinitionCondition{
-		Type:    apiextensions.Established,
+	// set InstallingCondition to true if all names are accepted. Never set it back to false.
+	installingCondition := apiextensions.CustomResourceDefinitionCondition{
+		Type:    apiextensions.Installing,
 		Status:  apiextensions.ConditionFalse,
 		Reason:  "NotAccepted",
 		Message: "not all names are accepted",
 	}
 	if old := apiextensions.FindCRDCondition(in, apiextensions.Established); old != nil {
```

Comment:
should this be `Established` or `Installing`?

In general, what is the flow: is `Installing` set to false as soon as `Established` is true?

Reply:
Agree. But I think the implemented logic does not match yet. If there is already Installing=true, we should keep it (especially its timestamp).